### PR TITLE
 added new metrics origin mapping table to docs

### PR DESCRIPTION
**Pull request by ouracheal**
### Merge instructions

- [x] merge after review

### CODE REVIEW

**Comment by hestonhoffman** [comment]
Hi! I've added the Do Not Merge label to remove this from our queue. When you're done, remove that label to put the PR back in our queue :)

**Comment by ouracheal** [comment]
> Hi! I've added the Do Not Merge label to remove this from our queue. When you're done, remove that label to put the PR back in our queue :)

Thanks!

File: content/en/metrics/summary.md [hunk 1]
```diff
@@ -181,11 +181,41 @@ More aggregations are available should they be valuable to you. You can add or r
...
+| Events                  | Custom metrics generated from the Datadog Events platform.
+| LLM Observability       | Timeseries emitted by the LLM Observability product via `lmobs_to_metrics` service.
+| Logs                    | Custom metrics generated from the Datadog [Logs][28] platform.
+| Metrics API             | Timeseries sent via Datadog's [OTLP Ingestion endpoint][21] and OTel receiver with a Datadog integration counterparts or points for estimated usage metrics or Datadog API Client.
+| NPM                     | Timeseries sent by the Datadog [Network Performance Monitoring][19] product.
+| Observability Pipelines | Timeseries sent by the Datadog [Observability Pipielines][20] including error and performance metrics.
+| Other                   | Custom metrics that don’t have a DD integration counterpart.
+| Processes               | Custom metrics generated from the Datadog [Processes][22] product.
+| RUM                     | Custom metrics generated from the Datadog [Real User Monitoring][23] product.
+| SAAS Integrations       | Timeseries collected from popular SAAS platforms like Slack, Docker, PagerDuty etc.
```
Comment (brett0000FF) [inline content/en/metrics/summary.md:208]:
```suggestion
| SaaS Integrations       | Timeseries collected from popular SaaS platforms like Slack, Docker, PagerDuty, etc.
```

**Review by brett0000FF — CHANGES_REQUESTED** [review]
Thanks for adding this comment! Please let me know if you have any questions about my suggestions. Most of them are addressing style tweaks that Vale flagged.

**Review by ouracheal — COMMENTED** [review]
added changes!

**Review by brett0000FF — APPROVED** [review]
🚀

**Pull request merged**